pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1665,14 +1663,14 @@ def label(self, decimals=None, base_label=None):
 
     def adjoint(self):
         return MultiControlledX(
-            control_wires=self.wires[:-1],
+            n_control_wires=len(self.wires[:-1]),
```

Comment:
here, this doesn't work because `__init__` still accepts `control_wires`

Reply:
Sorry! 

But As far as I can see, if we revert the change in this line things should work? Since adjoint can pass self.wires to `MultiControlledX`, it is not restricted like the static methods.